tutorials/quick_start/README.md

```diff
@@ -15,6 +15,159 @@ in the future.
 - ultralytics: [https://ultralytics.com](https://ultralytics.com)
 - keras-applications: [https://keras.io/api/applications](https://keras.io/api/applications/)
```

Comment:
1. Move the "Supported libraries" section below the table.
2. Add new section "Supported Features" which includes the following:
 - Quantize pretrained models from various model libraries [link to "Supported libraries] by one click
 - Evaluate the performance of the floating point model and the quantized model.
 - Use MCT's Post-Training Quantization (PTQ) scheme [link to example]
 - Use MCT's advanced quantization techniques (such as GPTQ and Mixed Precision) [link to the example]
3. Add section title: "Results" before the table.

"" (

Reply:
do we have an example of gptq in quick start?

Reply:
Scroll down this page, you will see: "Advanced model quantization example"